ui-tests/cypress/support/pageObjects/users/management/ManagementPage.js

```diff
@@ -123,12 +123,12 @@ class UsersKebabMenu extends KebabMenu {
 
   getManageAuth() {
     return this.get()
-               .find(`[${DATA_ID}=manageAuth-${this.code}]`);
+               .find(`${htmlElements.li}.UserListMenuAction__menu-item-auth`);
```

Comment:
Same 👆🏼

Reply:
I retrieved the cypress documentation where they clarify how functions like .click() and .type() work.
The action in the Dom generates some sort of context where the visibility of the element is checked by cypress before performing any actions.

Also, I did some checks to exclude a random match of the randomly generated element in the test.

Therefore it seems to me that the modification does not affect the purposes of the test.